firefox_ios/views/unified_metrics.view.lkml

```diff
@@ -0,0 +1,755 @@
+# Un-hide and use this explore, or copy the joins into another explore, to get all the fully nested relationships from this view
+explore: unified_metrics {
+  hidden: no
+
```

Comment:
Ah shoot, one last thing. Generally it's good practice to put the explores in the `model` file, and leave the `view` file with just the view definitions (this is general Looker guidance, but tbh it's sometimes more confusing b/c the understanding the views _requires_ seeing the explore, and vice-versa. So maybe they should just live in one file.)

Reply:
I'm going to leave this inside the view file for the meanwhile.